interfaces/Portalicious/src/app/pages/project/project-team/add-user-form/add-user-form.component.ts

```diff
@@ -50,23 +52,49 @@ type AddUserToTeamFormGroup =
     ReactiveFormsModule,
   ],
 })
-export class AddUserButtonComponent {
+export class AddUserFormComponent {
   projectId = input.required<number>();
-  enableScope = input.required<boolean | undefined>();
+  enableScope = input.required<boolean>();
+  formVisible = model.required<boolean>();
+  userToEdit = input<ProjectUserWithRolesLabel | undefined>();
 
   private projectApiService = inject(ProjectApiService);
   private userApiService = inject(UserApiService);
   private roleApiService = inject(RoleApiService);
   private toastService = inject(ToastService);
 
+  constructor() {
+    effect(
```

Comment:
This is basically how you would expect/_want_ the `ngOnChanges`-thing to work...

Still a bit magical. ;)